.annotation_safe_list.yml

```diff
@@ -7,6 +7,8 @@
 # fake_app_2.FakeModel2:
 #    ".. choice_annotation:": foo, bar, baz
 
+canvas.HistoricalCanvasEnterpriseCustomerConfiguration:
```

Comment:
didn't realize we had to add this too, thanks for covering this, but briefly, what does this do? I remember seeing it then later somehow we never added this entry

Reply:
I think you'll find it a little clearer on the next line (though it is still obviously vague). We have some test that requires some sort of annotation or tests or something if you have classes/methods that contain personally identifiable information (ie email addresses, addresses, social security number, etc.) so you have to specify here what the pii is, or alternatively that you don't use any.

Side note: I thought Alex actually copied the Canvas portion over to a Canvas PR because it was failing coverage tests, but perhaps not since it's showing as a change still